samples/NativeModuleSample/cppwinrt/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "native-module-sample",
+  "name": "native-module-sample-cppwinrt",
```

Comment:
If you rename this, when following the upgrade steps in the readme, the solution file, vcxproj project file, and namespaces will all be different.